edx_sysadmin/templates/edx_sysadmin/base.html

```diff
@@ -10,12 +13,12 @@
             <h1 class="main-heading">{% trans "Sysadmin Dashboard" %}</h1>
             <hr />
             <h2 class="instructor-nav">
-                <a href="#">{% trans "Users" %}</a>
+                <a href="{% url 'sysadmin:sysadmin_users' %}">{% trans "Users" %}</a>
```

Comment:
@HamzaIbnFarooq If we are using `sysadmin` namespace then we don't need to prefix url with sysadmin. This can be simplified like this
```
{% url 'sysadmin:users' %}
{% url 'sysadmin:courses' %}
```

Reply:
updated.